quantum/eeconfig.c

```diff
@@ -46,7 +46,7 @@ void eeconfig_init_quantum(void) {
     eeprom_update_byte(EECONFIG_DEFAULT_LAYER, 0);
     default_layer_state = 0;
     eeprom_update_byte(EECONFIG_KEYMAP_LOWER_BYTE, 0);
-    eeprom_update_byte(EECONFIG_KEYMAP_UPPER_BYTE, 0xC);
+    eeprom_update_byte(EECONFIG_KEYMAP_UPPER_BYTE, 0x14);
```

Comment:
Just to make this clear. 
```suggestion
    // Enable oneshot and autocorrect by default: 0b0001 0100
    eeprom_update_byte(EECONFIG_KEYMAP_UPPER_BYTE, 0x14);
```